### PR TITLE
Support for play videos in imagemap messages

### PR DESCRIPTION
**Pull request by moririnson**
Fixed https://github.com/line/line-bot-sdk-php/issues/191 .

### CODE REVIEW

File: src/LINEBot/MessageBuilder/Imagemap/VideoBuilder.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    /** @var ExternalLinkBuilder */
+    private $externalLinkBuilder;
+    
+    /**
+     * VideoBuilder constructor.
+     *
+     * @param string $originalContentUrl
+     * @param string $previewImageUrl
+     * @param AreaBuilder $area
+     * @param ExternalLinkBuilder $externalLink
```
Comment (satosby) [inline src/LINEBot/MessageBuilder/Imagemap/VideoBuilder.php]: Please fix to ```ExternalLinkBuilder|null```

Comment (moririnson) [reply src/LINEBot/MessageBuilder/Imagemap/VideoBuilder.php]: fixed!

File: examples/KitchenSink/src/LINEBot/KitchenSink/EventHandler/MessageHandler/TextMessageHandler.php [hunk 2]
```diff
@@ -174,6 +176,40 @@ public function handle()
                 );
                 $this->bot->replyMessage($replyToken, $imagemapMessageBuilder);
                 break;
+            case 'imagemap video':
```
Comment (Vaduz) [inline examples/KitchenSink/src/LINEBot/KitchenSink/EventHandler/MessageHandler/TextMessageHandler.php]: `quickReply` is snake case. So this also should be in the same manner?

Comment (moririnson) [reply examples/KitchenSink/src/LINEBot/KitchenSink/EventHandler/MessageHandler/TextMessageHandler.php]: change it to camel case!

**Pull request merged**